crates/codegen/parser/runtime/src/cst.rs

```diff
@@ -6,12 +6,28 @@ use crate::cursor::Cursor;
 use crate::kinds::{RuleKind, TokenKind};
 use crate::text_index::TextIndex;
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize)]
+pub struct NamedNode {
+    pub name: String,
+    pub node: Node,
+}
+
+impl NamedNode {
+    /// Creates an anonymous (nameless) node.
+    pub fn anon(node: Node) -> Self {
```

Comment:
I suggest using the full name, as it is only a few characters. I think `anon` also has other meanings: https://www.merriam-webster.com/dictionary/anon

Reply:
I haven't heard of these; looks archaic? I've changed it in https://github.com/NomicFoundation/slang/pull/710/commits/3b7d5dc20c0ff2f6dd62dc811fc8e559a064fa16